homework-classes/ContributorsView.js

```diff
@@ -19,8 +19,46 @@
      * @param {Object[]} contributors An array of contributor objects
      */
     render(contributors) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('ContributorsView', contributors);
+      const contributorsContainer = document.querySelector(
+        '.contributors-container',
+      );
+      const select = document.querySelector('.repo-select');
```

Comment:
Handling the `select` element is the responsibility of the `HeaderView` class. You should not reference it here.